hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/strategy/ClusteringPlanStrategy.java

```diff
@@ -104,6 +104,13 @@ public ClusteringPlanStrategy(HoodieTable table, HoodieEngineContext engineConte
    */
   public abstract Option<HoodieClusteringPlan> generateClusteringPlan();
 
+  /**
+   * Check if the clustering can proceed. If not, empty plan will be returned to stop the scheduling.
+   */
+  public boolean checkPrecondition() {
```

Comment:
the description is not aligned to the method.

Reply:
Thanks, will update.

Reply:
Thanks, will update,